app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -312,8 +331,12 @@ class TabTrayView(
         components.analytics.metrics.track(eventToSend)
     }
 
-    fun expand() {
-        behavior.state = BottomSheetBehavior.STATE_EXPANDED
+    fun accomodateTabs() {
+        if (getTabsNumberInAnyMode() >= getTabsNumberForExpandingTray() || isInLandscape()) {
```

Comment:
Let's flip the order of the `||` and have `isInLandscape()` first to allow the if statement to escape quicker if that one condition is true.

Reply:
Thanks! Will change it.
I was going with the presumption that most of the time the device will be in portrait but have not data to back this up.